### PR TITLE
add more options to html-to-text

### PR DESCRIPTION
**Pull request by mystvearn147**
add 4 more optional properties to HtmlToTextOptions interface
noLinkBrackets, preserveNewlines, uppercaseHeadings and singleNewLineParagraphs

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/werk85/node-html-to-text#options
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by erykwarren — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR appears ready to merge pending express review by a maintainer.

**Pull request merged**
